pkg/cluster/constants/kafka/kcManager.go

```diff
@@ -0,0 +1,52 @@
+package kafka
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	AKCGroup   = "rhoas.redhat.com"
+	AKCVersion = "v1alpha1"
+)
+
+var AKCRMeta = metav1.TypeMeta{
+	Kind:       "KafkaConnection",
+	APIVersion: AKCGroup + "/" + AKCVersion,
+}
+
+var AKCResource = schema.GroupVersionResource{
+	Group:    AKCGroup,
+	Version:  AKCVersion,
+	Resource: "kafkaconnections",
+}
+
+var tokenSecretName = "rh-cloud-services-accesstoken-cli"
```

Comment:
[ResponsibilityLeak]
This should not be part of the class. 
Token secret is happening before service is picked up and it should be shared